If PT_LOGGING_ENABLED or PT_DEBUG_PCI_CONFIG_ACCESS are set for debugging,
the unused variables related to these patches will cause the build to
fail.

Signed-off-by: Eric Shelton <eshelton@pobox.com>

--- a/tools/qemu-xen-traditional/hw/pass-through.c	2016-10-26 23:59:18.862265294 -0400
+++ b/tools/qemu-xen-traditional/hw/pass-through.c	2016-10-26 23:59:37.843265294 -0400
@@ -1796,7 +1796,7 @@
     {
         assigned_device->permissive_warned = 1;
         PT_LOG("Write-back to unknown field 0x%02x (partially) inhibited (0x%0*x)\n",
-               addr, len * 2, wb_mask);
+               address, len * 2, wb_mask);
         PT_LOG("If device %02x:%02x.%o doesn't work, try enabling permissive\n",
                pci_bus_num(d->bus), PCI_SLOT(d->devfn), PCI_FUNC(d->devfn));
         PT_LOG("mode (unsafe) and if it helps report the problem to xen-devel\n");
--- a/tools/qemu-xen-traditional/hw/pt-graphics.c	2016-10-28 06:43:16.854551807 -0400
+++ b/tools/qemu-xen-traditional/hw/pt-graphics.c	2016-10-28 06:49:50.644551807 -0400
@@ -91,7 +91,7 @@
     }
 #ifdef PT_DEBUG_PCI_CONFIG_ACCESS
     PT_LOG_DEV((PCIDevice*)real_dev, "addr=%x len=%lx val=%x\n",
-            PCI_INTEL_OPREGION, len, val);
+            PCI_INTEL_OPREGION, 4, val);
 #endif
 
 }
